### DISCUSSION TITLE
Copying difficulty deleted the old one

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18848] [Q&A]
cannot recreate with minimal testing.

[logs.zip](https://github.com/ppy/osu/files/8986096/logs.zip)

i made a copy of a difficulty, deleted everything in the copy, and then everything is gone in the original too. 

![image](https://user-images.githubusercontent.com/78441708/175802956-6fa33494-29e8-4f1b-bd6b-56d1a3666c3b.png)
i assure you that Axilotl's Nervous is was not 641 bytes before everything got yoinked. 

arch linux, osu! version 2022.621.0

**Comment by peppy** [comment]
How are you seeing the files in that screenshot? Is that after a package export?
